### PR TITLE
image-badge-navigation added

### PR DESCRIPTION
**Pull request by Shinde-Sandesh**

### CODE REVIEW

File: Components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+.avatar-xl {
+    height: 4em;
+    width: 4em;
+}
+
+.avatar-lg {
+    height: 3em;
+    width: 3em;
+}
+
```
Comment (pankaj-prog) [inline Components/badge/badge.css:72]: applying fixed height and width will  stretch the image if the image resolution is not square

File: Components/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+.online {
+    color: var(--green-color);
+}
+
+.offline {
+    color: var(--red-color);
+}
+.dnd {
+    color: var(--gray-color);
+}
```
Comment (pankaj-prog) [inline Components/badge/badge.css:94]: these colors are not getting applied, also use uppercase letters for hard coded value

File: Components/navigation/navigation.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
+.navigation-navbar-container{
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      padding: 1rem 2rem;
```
Comment (pankaj-prog) [inline Components/navigation/navigation.css:5]: be consistent with the units, somewhere you are using px in margin and padding whereas somewhere rem.

File: Components/navigation/navigation.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            <a href="/badge/badge.html">Badge</a>
+            <a href="/button/button.html">Button</a>
+            <a href="/card/card.html">Card</a>
+            <a href="/image/image.html">Image</a>
+            <a href="/input/input.html">Input</a>
+            <a href="/text-utilities/text-utilities.html">Text-Utilities</a>
+            <a href="/list/list.html">List</a>
+            <a href="#" class="focus">Navigation</a>
+      </div>   
+      <div class="main">
```
Comment (pankaj-prog) [inline Components/navigation/navigation.html:27]: use semantic tag main

File: Components/image/image.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+            <a href="/badge/badge.html">Badge</a>
+            <a href="/button/button.html">Button</a>
+            <a href="/card/card.html">Card</a>
+            <a href="#" class="focus">Image</a>
+            <a href="/input/input.html">Input</a>
+            <a href="/text-utilities/text-utilities.html">Text-Utilities</a>
+            <a href="/list/list.html">List</a>
+            <a href="/navigation/navigation.html">Navigation</a>
+      </div>   
+      <div class="main">
```
Comment (pankaj-prog) [inline Components/image/image.html:25]: use semantic tag main

File: Components/image/image.html [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+<head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Component-Library</title>
+      <link rel="stylesheet" href="/styles.css">
+      <link rel="stylesheet" href="image.css">
+</head>
+<body>
+      <div class="sidenav">
```
Comment (pankaj-prog) [inline Components/image/image.html:12]: use nav tag here

File: Components/image/image.css [hunk 7]
```diff
@@ -0,0 +1,34 @@
+.VariantContainer
+{
+      margin-top: 25px;
+      border: 3px solid var(--primary-color);
+      width: 900px;
```
Comment (pankaj-prog) [inline Components/image/image.css:5]: correct the indentation

File: Components/badge/badge.css [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      gap: 25px;
+      justify-content: center;
+      align-items: center;  
+      text-align: center; 
+}   
+
+#badge-container p span{
```
Comment (pankaj-prog) [inline Components/badge/badge.css:16]: id's specificity is too high, use it only when you donot have any other option, always keep the specificity as low as possible

**Review by pankaj-prog — CHANGES_REQUESTED** [review]

File: Components/badge/badge.css [hunk 9]
```diff
@@ -0,0 +1,94 @@
+.VariantContainer
+{
+      margin-top: 25px;
+      border: 3px solid var(--primary-color);
+      width: 900px;
+      height: 200px;
```
Comment (manojsarna) [inline Components/badge/badge.css:6]: container fixed height , width is not recommended for responsiveness use rems!!

can use rems for margin

File: Components/badge/badge.css [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      gap: 25px;
+      justify-content: center;
```
Comment (manojsarna) [inline Components/badge/badge.css:11]: can use rems for this!!

File: Components/badge/badge.css [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+#badge-container p span{
+    background-color: var(--green-color);
+    border-radius: 5px;
+    padding: .1rem;
+}
+
+.icons{
+    display: flex;
+    justify-content: center;
+    gap: 40px;
```
Comment (manojsarna) [inline Components/badge/badge.css:25]: check for px values over here

File: Components/image/image.css [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+  .image-responsive{
+    width: 100%;
+    height: auto;
+    object-fit: cover; 
+  
+  }
+  
+  .image-circle{
+      width: 140px;
+      height: 140px;
```
Comment (manojsarna) [inline Components/image/image.css:25]: dont use px values !!

File: Components/image/image.html [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+            <a href="/alert/alert.html">Alert</a>
+            <a href="/badge/badge.html">Badge</a>
+            <a href="/button/button.html">Button</a>
+            <a href="/card/card.html">Card</a>
+            <a href="#" class="focus">Image</a>
+            <a href="/input/input.html">Input</a>
+            <a href="/text-utilities/text-utilities.html">Text-Utilities</a>
+            <a href="/list/list.html">List</a>
+            <a href="/navigation/navigation.html">Navigation</a>
+      </div>
```
Comment (manojsarna) [inline Components/image/image.html:24]: use semantics html over here

File: Components/image/image.html [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+            <div>
+                  <h3>Responsive Image</h3>
+          
+                  <div>
+                        <img class="image-responsive" src="2550833-bigthumbnail.jpg" alt="image3">
+                  </div>
+                  <h3>Round Image</h3>
+                  <div class="VariantContainer no-border">
+                      <img class="image-circle" src="2550833-bigthumbnail.jpg" alt="image2">
+                  </div>
```
Comment (manojsarna) [inline Components/image/image.html:36]: dont use jpg , png , you can use webp instead for 

- Better Performance of the website
- Compressed Size
- Good Maintained Quality of the images

File: Components/navigation/navigation.css [hunk 15]
```diff
@@ -0,0 +1,52 @@
+.navigation-navbar-container{
+      display: flex;
+      justify-content: center;
+      align-items: center;
```
Comment (manojsarna) [inline Components/navigation/navigation.css:4]: use Beautify to align your code

File: Components/navigation/navigation.html [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+            <a href="/alert/alert.html">Alert</a>
+            <a href="/badge/badge.html">Badge</a>
+            <a href="/button/button.html">Button</a>
+            <a href="/card/card.html">Card</a>
+            <a href="/image/image.html">Image</a>
+            <a href="/input/input.html">Input</a>
+            <a href="/text-utilities/text-utilities.html">Text-Utilities</a>
+            <a href="/list/list.html">List</a>
+            <a href="#" class="focus">Navigation</a>
+      </div>
```
Comment (manojsarna) [inline Components/navigation/navigation.html:26]: use semantics nav , aside , main , header , footer

**Review by manojsarna — CHANGES_REQUESTED** [review]
Do Check the comments

Do check your website there is something broken, cant see these changes you have made!!!

**Pull request merged**